pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -75,7 +90,8 @@ def solve(self, model, **kwds):
         Args:
             model (Block): a Pyomo model or block to be solved
         """
-        config = self.CONFIG(kwds.pop('options', {}))
+        config = self.CONFIG(kwds.pop('options', {
+        }), preserve_implicit=True)  # TODO: do we need to set preserve_implicit=True?
```

Comment:
I do not know where is this coming from. Can you tell me why are we using this?

Reply:
This function helps MindtPy obtain all the options. In GDPOpt, Qi set preserve_implicit=True.